docs/sources/enterprise/saml.md

```diff
@@ -109,6 +117,55 @@ Grafana provides configuration options that let you modify which keys to look at
 
 An example is `assertion_attribute_name = "givenName"` where Grafana looks within the assertion for an attribute with a friendly name or name of `givenName`. Both, the friendly name (e.g. `givenName`) or the name (e.g. `urn:oid:2.5.4.42`) can be used interchangeably as the value for the configuration option.
 
+### Configure team sync
+
+> Only available in Grafana v7.0+
+
+To be able to use Team Sync feature, set `assertion_attribute_groups` option to the attribute name where you store user groups. Then Grafana will use attribute values extracted from SAML assertion to add user into the groups with the same name configured on the External group sync tab.
+
+[Learn more about Team Sync]({{< relref "../enterprise/team-sync.md" >}})
+
+### Configure role sync
+
+> Only available in Grafana v7.0+
+
+Role sync allows to map user roles from identity provider to Grafana. To enable role sync, set `assertion_attribute_role` option to the attribute name where the role information will be extracted from. After that, configure desired attribute values for each of Grafana roles: Editor, Admin and Super Admin. If obtained role doesn't match any of configured values, `Viewer` role will be assigned to the user. Example configuration:
+
+```bash
+[auth.saml]
+assertion_attribute_role = role
+role_values_editor = editor, developer
+role_values_admin = admin, operator
+role_values_grafana_admin = superadmin
+```
+
+Refer to [Organization roles]({{< relref "../permissions/organization_roles.md" >}}) for more information about roles and permissions in Grafana.
+
+**Important**: when role sync is configured, any changes of user roles and organization membership made manually in Grafana will be overwritten on next user login. Assign user organizations and roles in the IdP instead.
```

Comment:
```suggestion
**Important**: When role sync is configured, any changes of user roles and organization membership made manually in Grafana will be overwritten on next user login. Assign user organizations and roles in the IdP instead.
```

Reply:
Where is the IdP? What does this acronym stand for?

Reply:
At the top of the page we have
>SAML authentication integration allows your Grafana users to log in by using an external SAML 2.0 Identity Provider (IdP)